src/components/ResultsList.vue

```diff
@@ -15,6 +15,38 @@ import ResultsListItem from './ResultsListItem.vue';
 import FindPark from './FindPark.vue';
 export default {
   props: ['searchResults'],
-  components: { ResultsListItem, FindPark }
+  data() {
+    return {
+      noResults: false
+    }
+  },
+  methods: {
+    load() {
+      if (!this.$store.state.searchResults.length) {
+        this.noResults = true
+      }
+    }
+  },
+  mounted() {
+    setTimeout(() => this.load(), 6000)
+  },
+  components: { ResultsListItem, FindPark },
+  computed: {
+    message() {
+      if (this.noResults) {
+        return 'Sorry, no results found. Try a different search!'
+      } else if (!this.noResults && !this.$store.state.searchResults.length) {
+        return 'Loading parks...'
+      } else {
+        return ''
+      }
+    }
+  }
 }
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+.loading {
+  cursor:  wait;
+}
```

Comment:
This is exactly what I would have done in terms of the cursor's appearance and location. You have the loading class in the parent most element, so I would think it would appear anywhere in that section, not just over the text. Maybe the overall cursor property only exists when it is positioned over some sort of feature?

Reply:
This css feature only persist for the cursor. Most of the spinning icons you find in website are custom divs made with css and animations. I will have a more consistent loading icon with the next update before tackling desktop styling.